index.js

```diff
@@ -1,25 +1,52 @@
 'use strict';
 
-module.exports = function buildAirbnbPreset(context, options) {
-  var preset; // eslint-disable-line no-var
-  if (options && options.modules === false) {
-    preset = require('babel-preset-es2015').buildPreset(null, { modules: false });
-  } else {
-    preset = require('babel-preset-es2015-without-strict');
+const browsers = [
+  'Chrome >= 35',
+  'Explorer >= 9',
+  'Firefox >= 52',
+  'Safari >= 8',
+];
+
+function setNodeVersion(version) {
+  if (typeof version !== 'string' && version !== true) {
+    throw new TypeError('`node` must either be `true` or a string');
   }
+  return version === true ? '6.10.2' : version;
```

Comment:
i don't think we should hardcode 6.10.2 anywhere. We shouldn't need to bump babel-preset-airbnb every time we upgrade node internally, and this version isn't useful for anyone besides us.

iow, I think it should just always be a semver string (and we may want to use the `semver` module to validate it)